source/Core/Threads/GUIThread.cpp

```diff
@@ -553,9 +553,9 @@ static void gui_solderingMode(uint8_t jumpToSleep) {
 #else
     if (OLED::getRotation()) {
 #endif
-      OLED::setCursor(50, 0);
```

Comment:
What was this for?

Reply:
Sorry, that was me not realizing these changes would also be pulled. 😓